src/AltinnCore/Common/Languages/LanguageController.cs

```diff
@@ -0,0 +1,68 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.IO;
+using IniParser;
+using IniParser.Model;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using AltinnCore.Common.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace AltinnCore.Designer.Controllers
+{
+    public class LanguageController : Controller
+    {
+        private readonly GeneralSettings _generalSettings;
+        public LanguageController(
+            IOptions<GeneralSettings> generalSettings)
+        {
+            _generalSettings = generalSettings.Value;
+        }
+        /// <summary>
+        /// Reads ini files, converts it to json
+        /// </summary>
+        /// <param name="org">The current service owner</param>
+        /// <param name="service">The current service</param>
+        /// <param name="languageCode">The current language code</param>
+        /// <returns>The model representation as JSON</returns>
+        [HttpGet]
+        public IActionResult getLanguageAsJSON(string org, string service, string languageCode)
+        {
+            var parser = new FileIniDataParser();
```

Comment:
Don't use "var" in C# makes it hard to understand the type of the variable

Reply:
no vars any more